micromamba/tests/test_env.py

```diff
@@ -363,3 +364,44 @@ def test_env_update_pypi_with_conda_forge(tmp_home, tmp_root_prefix, tmp_path):
     pip_packages_list = yaml.safe_load(pip_list_output)
 
     assert any(pkg["name"] == "numpy" and pkg["version"] == "1.26.4" for pkg in pip_packages_list)
+
+
+@pytest.mark.parametrize("shared_pkgs_dirs", [True], indirect=True)
+def test_env_create_whitespace(tmp_home, tmp_root_prefix, tmp_path):
```

Comment:
I don't think we need this test, tests in `test_match_spec` are enough to make sure the parsing is done correctly.
Adding this one is redundant and is not relevant in my opinion.

Reply:
I agree with this, I just wanted to have a non-regression test for the reported issue.